gatsby-node.esm.js

```diff
@@ -308,9 +317,18 @@ exports.createPages = async ({ actions, graphql }) => {
             },
           })
         })
+      } else {
+        // No need to create referrer URLs 
+        createPage({
+          path: `${path}/${data.causeId}/`,
```

Comment:
For more external readability, let's do `${path}/${node.path}/preview/`. This wasn't in the spec, sorry!